seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseFileSinkConfig.java

```diff
@@ -47,6 +47,10 @@ public class BaseFileSinkConfig implements DelimiterConfig, Serializable {
     protected DateTimeUtils.Formatter datetimeFormat = DateTimeUtils.Formatter.YYYY_MM_DD_HH_MM_SS;
     protected TimeUtils.Formatter timeFormat = TimeUtils.Formatter.HH_MM_SS;
     protected Boolean enableHeaderWriter = false;
+    protected Boolean enableHiveCollectionType =
```

Comment:
Hive config can not put in `BaseFileSinkConfig`.

I think determining how to write a file through the upstream field types `Array, Map, Row` is a general function rather than a Hive-specific feature.